application/lib/main.dart

```diff
@@ -44,19 +44,19 @@ class CuddlyWorldIDE extends StatelessWidget {
 }
 
 class MainScreen extends StatefulWidget {
-  const MainScreen({ Key key }) : super(key: key);
+  const MainScreen({ Key? key }) : super(key: key);
 
   @override
   _MainScreenState createState() => _MainScreenState();
 }
 
 class _MainScreenState extends State<MainScreen> with SingleTickerProviderStateMixin {
-  CuddlyWorld _game;
-  Terminal _terminal;
-  StreamSubscription<String> _gameStream;
-  TabController _tabController;
+  CuddlyWorld? _game;
+  late final Terminal _terminal;
+  StreamSubscription<String>? _gameStream;
+  TabController? _tabController;
```

Comment:
_terminal and _tabController are initialized at the same time so let's make them consistent

Reply:
fixed